main/auth/courses.php

```diff
@@ -156,8 +156,9 @@
 }
 // We are unsubscribing from a course (=Unsubscribe from course).
 if (isset($_GET['unsubscribe'])) {
+    $search_term = isset($_GET['searh_term']) ? intval($_GET['searh_term']) : null;
```

Comment:
Wow, i dont know why that intval() function is there, uhmm there is no "c" variable but this fixes the fact that when you unsubscribe from the main page (index.php - hot courses)  doesn't have a search function there and for that the var search_term doesn't pass for $_GET, only verify if search_term exist